docs/ref/utils.txt

```diff
@@ -556,6 +556,36 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
         z = person.friends         # does not call
         x is z                     # is True
 
+.. class:: classproperty
+
```

Comment:
`versionadded` is missing:
```
.. class:: classproperty(method=None)

.. versionadded:: 3.1
```

Reply:
At all other places, `versionchanged` / `versionadded` annotations were indented as if they were "inside" the code block, so in order to maintain consistency, I have kept this indented too "inside" `class:: classproperty(method=None)` block as follows, please let me know if this is not how it is supposed to be done.

```
.. class:: classproperty(method=None)

    .. versionchanged:: 3.1
```